src/Student5/Ex1/DateFetcher.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useState } from "react";
+
+export const DateFetcher = () => {
+ 
```

Comment:
Let's make this look nice, remove the whitespace and indent properly.

Reply:
white spaces have been removed